CONTRIBUTING.md

```diff
@@ -0,0 +1,114 @@
+
+If you have any questions or want to read more, check out the [18F Open Source Policy GitHub repository]( https://github.com/18f/open-source-policy), or [send us an email](mailto:18f@gsa.gov).
+
+
+## Pull request guidelines
+
+Below are rules we strive to follow to achieve maintainable and consistent code.
+
+### Commit message style guide:
+
+- Write your commit message summary in the imperative: "Fix bug" and not
+"Fixed bug" or "Fixes bug."  This convention matches up with commit messages
+generated by commands like `git merge` and `git revert`.
+
+- Under the summary, start by explaining why this change is necessary, and
+add details to help the person reviewing your code understand what your
+pull request is about.
+
+- If the pull request is in response to a Jira ticket, include the ticket ID in
+the commit title (e.g. "LG-1234 Add the stuff to the thing")
+
+- Include a changelog message which describes the changes in human-readable terms. Refer to the
+[_Changelog Messages_ section](#changelog-messages) below for specific changelog requirements.
+
+Example:
+
+```
+LG-1235 Load seed using before(:suite) in RSpec config
+
+**Why**:
+- Loading the seed in a `before(:each)` block results in an unnecessary
+database call before every single test, slowing down the test suite,
+and making development less efficient.
+
+**How**:
+- Use `before(:suite)` instead, since the data that is loaded is not
+meant to change, and so that only one database call is made.
+- To prevent the data from being wiped out after each spec, configure
+Database Cleaner to ignore those static tables.
+
+changelog: Internal, Automated Testing, Improve performance of test suite
+```
+
+#### Changelog Messages
+
+You must include a changelog message in one commit of your pull request. The changelog message
+describes the changes in human-readable terms. These messages are included in release notes, so they
+should be easy to understand for our partners and users.
+
+A changelog message should be written in the following format:
+
+```
+changelog: [Category], [Subcategory], [Description]
+```
+
+Replace `[Category]`, `[Subcategory]`, and `[Description]` with text relevant for your changes:
+
+- **Category** must be one of the following:
+   - **User-Facing Improvements** are improvements to the application experience benefitting the end-user, such as a new UI component or updated text.
+   - **Bug Fixes** are corrections to a broken behavior, such as preventing a raised exception.
+   - **Internal** are changes which benefit the Login.gov team, such as analytics or code quality.
+   - **Upcoming Features** are iterations contributing to a feature which has not yet been enabled for users in production.
+- **Subcategory** does not have any restrictions, but you should try to maintain consistency with changesets affecting similar parts of the application (for example, "In-person proofing").
+- **Description** is a plain language description of the specific changes.
```

Comment:
how do we feel about implementing this changelog check? Are we planning to do release notes for this repo? My guess is maybe we could remove it